modules/virt-adding-templates-to-custom-namespace.adoc

```diff
@@ -0,0 +1,45 @@
+// Module included in the following assemblies:
+//
+// * virt/vm_templates/virt-adding-templates-to-custom-namespace.adoc
+
+[id="virt-adding-templates-to-custom-namespace_{context}"]
+= Adding templates to a custom namespace
+
+The `ssp-operator` deploys Red Hat templates to the `openshift` namespace by default. Templates in the `openshift` namespace are publicly availably to all users. To add templates to a custom namespace, edit the `hco` custom resource (CR) which contains the `ssp-operator`.
+
+.Procedure
+
+* View the list of Red Hat templates that are available in the `openshift` namespace.
++
+[source,terminal]
+----
+$ oc get templates -n openshift
+----
++
+* Open the `hco` custom resource (CR).
++
+[source,terminal]
+----
+$ oc edit hco
```

Comment:
It needs to specify the namespace `openshift-cnv` here.

Reply:
In the video meet that I had with @ksimon1 and when he walked through the steps, he explicitly mentioned that we are editing the hco custom resource. Therefore, I am asking him for clarification.

Reply:
@lmandavi Hi Lavanya, I just happened to see this; maybe I can help. if this is to edit the HCO custom resource, this is the command that we use in the other doc sections:

`$ oc edit hco -n openshift-cnv kubevirt-hyperconverged`

Reply:
`openshift-cnv` is for downstream and `kubevirt-hyperconverged` is for upstream.

Reply:
@gouyang - Please confirm - should this doc content be updated to show the command that @ousleyp mentioned above : `$ oc edit hco -n openshift-cnv kubevirt-hyperconverged ` ?

Reply:
> @gouyang - Please confirm - should this doc content be updated to show the command that @ousleyp mentioned above : `$ oc edit hco -n openshift-cnv kubevirt-hyperconverged ` ?

@lmandavi edit command is correct

Reply:
confirm it's correct!

Reply:
Updated the content.